x-pack/plugins/apm/server/lib/rum_client/get_client_metrics.ts

```diff
@@ -53,9 +63,11 @@ export async function getClientMetrics({
   // Divide by 1000 to convert ms into seconds
   return {
     pageViews,
-    backEnd: { value: (backEnd.value || 0) / 1000 },
+    backEnd: { value: (backEnd.values['50.0'] || 0) / 1000 },
```

Comment:
If you have time, can you update the return type for a percentiles aggregation? it's `Record<string, number>`, but it should be `Record<string, number | null>`.

Reply:
done